documentatie/use-cases/GSB-subfuncties.md

```diff
@@ -0,0 +1,62 @@
+# Use cases GSB - subfunctie,  vis 🐟
+
+## De invoerder handelt de fout(en) af
+
+__niveau:__ subfunctie, vis, 🐟
+
+### Hoofdscenario en uitbreidingen
+
+__trigger:__ De controles geven een foutmelding vanwege de [validatieregels voor fouten](./GSB-validatieregels-plausibiliteitschecks.md#validatieregels-geven-fouten).
+
+*Foutmelding*: De ingevoerde waardes kunnen niet correct zijn. Bijvoorbeeld: het totaal van de stemmen op een lijst komt niet overeen met de som van de stemmen van de kandidaten op die lijst.
+
+__hoofdscenario__:  
+Voor elke foutmelding:  
+
+1. De invoerder controleert de foutmelding.
+2. De invoerder constateert dat hij/zij een fout heeft gemaakt in de invoer.
+3. De invoerder corrigeert de fout in de invoer.
+
+__uitbreidingen__:  
+2a. De invoerder stelt een fout op het PV vast en kan de foutmelding niet oplossen:  
+&emsp; 2a1. De invoerder meldt de fout op het PV bij de coördinator.  
+&emsp; 2a2. De coördinator stuurt het PV terug in het proces.  
+&emsp; 2a3. De invoerder breekt de invoer af.  
+&emsp; 2a4. De applicatie verwijdert de ingevoerde data.  
+
+### Open punten
+
+- Als de coördinator het PV terugstuurt in het proces, naar welk punt dan precies?
+
+## De invoerder handelt de waarschuwing(en) af
+
+__niveau:__ subfunctie, vis, 🐟
+
+### Hoofdscenario en uitbreidingen
+
+__trigger:__ De controles geven een waarschuwing vanwege de [plausibiliteitschecks](./GSB-validatieregels-plausibiliteitschecks.md#plausibiliteitschecks-geven-waarschuwingen).
+
+*Waarschuwing*: De ingevoerde waardes zijn mogelijk niet correct. Bijvoorbeeld: er is een groot aantal blanco stemmen of de tweede invoer klopt niet met de eerste invoer.
+
+__hoofdscenario__:  
+Voor elke waarschuwing:  
+
+1. De invoerder controleert de waarschuwing.
+2. De invoerder constateert dat de invoer klopt met het PV.
+3. De invoerder meldt de waarschuwing bij de coördinator.
+4. De coördinator beslist dat de waarschuwing genegeerd kan worden.
+5. De invoerder accepteert de waarschuwing in de applicatie.
+
+__uitbreidingen__:  
+2a. De invoerder constateert dat hij/zij een fout heeft gemaakt in de invoer.  
+&emsp; 2a1. De invoerder corrigeert de fout in de invoer.  
+
+4a. De coördinator beslist dat de waarschuwing niet genegeerd kan worden:  
```

Comment:
Deze vervallen volgens mij ook bij de invoerder. @jschuurk-kr dit is in OSV nu toch ook zo? Of moet in OSV voor elke waarschuwing met de coordinator overlegt worden?

Reply:
OSV doet daar geen uitspraak over. Die laat de waarschuwing zien en laat de gebruiker toe verder te gaan. Proces daaromheen wordt niet beschreven of afgedwongen.

Wat in de praktijk bij invoer gebeurt (althans bij invoer bij de Kiesraad), is dat als een invoerder tegen een waarschuwing aanloopt, de invoerder de coördinator erbij roept. De coördinator beslist dan of het ok is de waarschuwing te accpeteren of dat er iets anders moet gebeuren.
Of dat is toch wat er gebeurt de eerste keer dat een invoerder tegen een bepaalde waarschuwing aanloopt. De tweede keer volg je gewoon het beslisproces van de vorige keer. Dus in dat opzicht is het erbij halen van de coörindator eerder een instructie-moment dan een beslismoment.

Het [proces met het dashboard](https://github.com/kiesraad/abacus/pull/293#discussion_r1743355059) is dan inderdaad een mooiere oplossing. Dan doen we iets nuttig in de software met alle waarschuwingen en invoerders kunnen altijd tussentijds vragen stellen aan de coördinator als ze dat willen.

Reply:
Als het meer instructie is, dan iets dat de software afdwingt, stel ik voor dat we we deze use cases hier verwijderen. En de stappen 3 en 4 bij bij [mijn eerdere comment](https://github.com/kiesraad/abacus/pull/293#discussion_r1743355059) ook. Eens?

Reply:
Eens. Moeten we wel gelijk de hoogover use case "De invoerders vullen de resultaten van de tellingen in" en mogelijk uitbreiding 3a van "Het GSB voert de PV's en eventuele SB corrigenda's (DSO) in de applicatie in" aanpassen met die controle na eerste invoer.

Reply:
Aangepast via twee commits
- Verwijderd bij invoerder
- Uitbreiding voor beoordelen waarschuwingen toegevoegd bij coordinator (in het midden gelaten of dat na eerste of tweede invoer gebeurd, dat maakt denk ik niet echt uit?)
- Je laatste opmerking: volgens mij hebben we die al, namelijk uitbreiding 3b: De applicatie stelt vast dat er stembureaus met waarschuwingen zijn

Reply:
Zoals het er nu staat, beoordeelt de coördinator de waarschuwingen na de tweede invoer.

Daar zit voor mij nog wel een proces-vraag achter: als je aanvullend onderzoek laat doen als GSB, er herteld wordt, en er een ander resultaat uitkomt, vervang je dan het PV (alleen een optie bij CSO) of doe je dan een corrigendum? In het eerste geval wil je dat de coördinator de waarschuwingen beoordeelt na de eerste invoer. Blijft de vraag in hoeverre we dat in de software moeten ondersteunen bijvoorbeeld dmv een aparte status.

Ik zou ook een aparte stap in het hoofdscenario maken dat er geen verschillen zijn en de beoordeling daar een uitbreiding van maken.

Bij Uitbreiding 3b: "De applicatie stelt vast dat er stembureaus met waarschuwingen zijn" vroeg ik me af of daar ook nog eens controleren op waarschuwingen zinvol is. Denk het wel. En anders kunnen we dat later aanpassen. Ik zie wel dat ik nog iets anders aan wil passen: expliciete stap in het hoofdscenario dat de applicatie controleert op waarschuwingen en niet ingevulde stembureaus.

Ik zal in een paar commits mijn suggesties doorvoeren. Kan ze altijd weer terugdraaien / verder aanpassen.

Reply:
96ce5e447eb891a2b02e754fa59a7dcaed42f8d6 en  ffda2c959f304cf202cab57420425c80c414cd34